libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -1980,7 +1999,9 @@
         "calculationId": "Beregnings-ID",
         "executionType": "Afviklingstype",
         "executionTime": "Afviklingstidspunkt",
-        "gridAreas": "Netområder"
+        "gridAreas": "Netområder",
+        "meteringPointType": "{{ meteringPoint.overview.shared.meteringPointType }}",
```

Comment:
question: is this necessary? It's found here but not in the English translation file.

Reply:
Added it to the english versino as well